runtime/runtime_storage.go

```diff
@@ -29,6 +30,8 @@ import (
 	"github.com/onflow/cadence/runtime/interpreter"
 )
 
+const ENCODING_NUM_WORKER = 10
```

Comment:
If you are thinking about a fallback, yes, setting to 1 works, it should follow the current behaviour, although it will generate more memory garbage, because of the slicing:

`bundle, newBatch := batch[:1], batch[1:]` 

Imaging there are 100 items in a batch, the above operation will repeat for 100 times, which is additional cost compare to the original version.

Reply:
Right, good point! I think this is good enough as a fallback in case we need it 👍